web_src/js/features/repo-issue.ts

```diff
@@ -150,6 +150,13 @@ export function initRepoIssueCommentDelete() {
           counter.textContent = String(num);
         }
 
+        const json: Record<string, any> = await response.json();
+        if (json.errorMessage) throw new Error(json.errorMessage);
+
+        if (json.deletedReviewCommentHashTag) {
+          document.querySelector(`#${json.deletedReviewCommentHashTag}`)?.remove();
+        }
```

Comment:
You can see that the lines around this uses the data attributes. So why here it uses a backend-generated ID?

https://github.com/go-gitea/gitea/pull/35103#discussion_r2217853784

<img width="1710" height="1036" alt="Image" src="https://github.com/user-attachments/assets/55d288ac-058c-40c6-905c-de7a3010d24b" />